src/projects/shared/api/circuit/src/internal/assembly/PrimBounds.ts

```diff
@@ -22,6 +22,9 @@ export function Bounds(prim: Prim): Rect {
         return prim.transform.asRect();
     case "SVG":
         return prim.transform.asRect();
+    case "Text":
+        // Text doesn't know its own size because that requires canvas
```

Comment:
Add to this comment saying something like "so make sure you have another prim surrounding it other it will get culled" or something like that